java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -326,6 +328,52 @@ void testReadJSONFile() {
       assertTablesAreEqual(expected, table);
     }
   }
+  @Test
+  void testReadMixedType2JSONFileFeatureDisabled() {
+    Schema schema = Schema.builder()
+            .column(DType.STRING, "a")
+            .build();
+    JSONOptions opts = JSONOptions.builder()
+            .withLines(true)
+            .withMixedTypesAsStrings(false)
+            .build();
+    assertThrows(CudfException.class, () ->
+      Table.readJSON(schema, opts, TEST_MIXED_TYPE_2_JSON));
+  }
+  @Test
```

Comment:
nit: add line to separate tests

```suggestion

  @Test
```